tests/shared/src/test/scala/scala/meta/tests/parsers/TermSuite.scala

```diff
@@ -337,6 +337,28 @@ class TermSuite extends ParseSuite {
     val Try(Lit(2), Nil, None) = term("try (2)")
   }
 
+  test("try (1 + 2).toString") {
+    val Try(
+      Select(
+        ApplyInfix(Lit(1), TermName("+"), Nil, Lit(2) :: Nil),
+        TermName("toString")
+      ),
+      Nil,
+      None
+    ) = term("try (1 + 2).toString")
+  }
+
+  test("try { 1 + 2 }.toString") {
+    val Try(
+      Select(
+        Term.Block(ApplyInfix(Lit(1), TermName("+"), Nil, Lit(2) :: Nil) :: Nil),
+        TermName("toString")
+      ),
+      Nil,
+      None
+    ) = term("try { 1 + 2 }.toString")
```

Comment:
It would be good to additionally test infix operators
```scala
        term("try (1 :: Nil) map fn")
```

Reply:
done